spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -755,6 +763,45 @@ def setup_and_check(self, data, title='', readonly=False,
         self.stack.currentChanged.connect(self.current_widget_changed)
         self.layout.addWidget(self.stack, 1, 0)
 
+        # ---- Toolbar and actions
+        toolbar = SpyderToolbar(parent=None, title='Editor toolbar')
+        toolbar.setStyleSheet(str(PANES_TOOLBAR_STYLESHEET))
+
+        self.copy_action = create_action(
+            self, _('Copy'), icon=ima.icon('editcopy'),
+            triggered=self.arraywidget.view.copy)
+        toolbar.addAction(self.copy_action)
```

Comment:
`SpyderToolbar` has `add_item`, so you can use here with `omit_id=True`.

Reply:
Done, though I left out the `omit_id=True` since that does not seem to be necessary now that I use `SpyderWidgetMixin.create_action()`.